addons/mail/static/src/discuss/call/common/meeting.js

```diff
@@ -35,6 +45,11 @@ export class Meeting extends Component {
         onMounted(() => (this.store.meetingViewOpened = true));
         onWillUnmount(() => (this.store.meetingViewOpened = false));
         useChildSubEnv({ inMeetingView: true });
+        useExternalListener(browser, "keydown", async (event) => {
+            if (event.key === "Escape" && !isEventHandled("NavigableList.close")) {
+                await this.rtc?.exitFullscreen();
+            }
+        });
```

Comment:
This doesn't work unless pressing a modifier, e.g. `Shift+Escape`. It should work with `Escape`, should investigate why this doesn't work?
Also should not close when interacting inside other panels, like invite people.
Opening message "expand" action and escape should only close message actions, not the meeting.

I wonder if all these cases are handled events? If so, maybe could be a good idea to have a utils function to know whether the event has been handled at least one, that could be the simplest condition to ignore handling it here.